instructor/templates/instructor/base.html

```diff
@@ -0,0 +1,188 @@
+<link type="text/css" href="/static/assignments/mit706s13/gen/instructions.gss.css?_=1417401991" rel="Stylesheet" />
+<link type="text/css" href="/static/mit706s13/gen/addmultipledialog.gss.css?_=1417401991" rel="Stylesheet" />
+<link type="text/css" href="/static/ui/gen/experiment_design.gss.css?_=1417401991" rel="Stylesheet" />
+<link type="text/css" href="/static/assignments/tufts/gen/instructions.gss.css?_=1417401991" rel="Stylesheet" />
+<link type="text/css" href="/static/assignments/mit706s14ps2/gen/instructions.gss.css?_=1417401991" rel="Stylesheet" />
+<link type="text/css" href="/static/coin-slider/coin-slider-styles.css?_=1417401991" rel="Stylesheet" />
+<link type="text/css" href="/static/css/smoothness/jquery-ui.min.css?_=1417401991" rel="Stylesheet" />
+<link type="text/css" href="/static/ui/gen/microscopy.gss.css?_=1417401991" rel="Stylesheet" />
+<link type="text/css" href="/static/css/jqdialog.css?_=1417401991" rel="Stylesheet" />
+<link type="text/css" href="/static/instructor/gen/instructor_experiment_setup.gss.css?_=1417401991" rel="Stylesheet" />
+<link type="text/css" href="/static/assignments/decusability/gen/instructions.gss.css?_=1417401991" rel="Stylesheet" />
+<link type="text/css" href="/static/ui/gen/homepage.gss.css?_=1417401991" rel="Stylesheet" />
+<link type="text/css" href="/static/ui_old/gen/experiment.gss.css?_=1417401991" rel="Stylesheet" />
+<link type="text/css" href="/static/ui/gen/assignments.gss.css?_=1417401991" rel="Stylesheet" />
+<link type="text/css" href="/static/starcellbio.app.css?_=1417401991" rel="Stylesheet" />
+<link type="text/css" href="/static/css/jquery.multiselect.css?_=1417401991" rel="Stylesheet" />
+<link type="text/css" href="/static/ui/gen/userguide.gss.css?_=1417401991" rel="Stylesheet" />
+<link type="text/css" href="/static/instructor/gen/instructor_assignment_setup.gss.css?_=1417401991" rel="Stylesheet" />
+<link type="text/css" href="/static/ui_old/gen/westernblotgel.gss.css?_=1417401991" rel="Stylesheet" />
+<link type="text/css" href="/static/microscopy_usability/gen/addmultipledialog.gss.css?_=1417401991" rel="Stylesheet" />
+<link type="text/css" href="/static/css/layout-default.css?_=1417401991" rel="Stylesheet" />
+<link type="text/css" href="/static/instructor/gen/instructor_course_setup.gss.css?_=1417401991" rel="Stylesheet" />
+<link type="text/css" href="/static/ui_old/gen/common.gss.css?_=1417401991" rel="Stylesheet" />
+<link type="text/css" href="/static/ui/gen/select_technique.gss.css?_=1417401991" rel="Stylesheet" />
+<link type="text/css" href="/static/ui/gen/auth.gss.css?_=1417401991" rel="Stylesheet" />
+<link type="text/css" href="/static/css/smoothness/jquery-ui-1.8.16.custom.css?_=1417401991" rel="Stylesheet" />
+<link type="text/css" href="/static/contact/contact.css?_=1417401991" rel="Stylesheet" />
+<link type="text/css" href="/static/ui_old/gen/experiment_setup.gss.css?_=1417401991" rel="Stylesheet" />
```

Comment:
also, as below, please use the staticfiles tag like is down for the javascript, i.e. `{% static "ui_old/gen/experiment...." %}`.  Also, I thought we nuked all that ui_old stuff?   This list seems crazy....